flink-python/pyflink/ml/lib/param/colname.py

```diff
@@ -0,0 +1,55 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+from pyflink.ml.api.param import WithParams, ParamInfo, TypeConverters
+
+
+class HasSelectedCols(WithParams):
+    """
+    An interface for classes with a parameter specifying the name of multiple table columns.
+    """
+
+    selected_cols = ParamInfo(
+        "selectedCols",
+        "Names of the columns used for processing",
+        is_optional=False,
+        type_converter=TypeConverters.to_list_string)
+
+    def set_selected_cols(self, v: list) -> 'HasSelectedCols':
+        return super().set(self.selected_cols, v)
+
+    def get_selected_cols(self) -> list:
+        return super().get(self.selected_cols)
+
+
+class HasOutputCol(WithParams):
```

Comment:
any reason why the output column has to be only 1? do we think this interface is sufficient moving forward?

Reply:
This is the behavior of the Java API. Maybe the reason is we use another interface, i.e., `HasOutputCols` to support multi-columns?